src/release.rs

```diff
@@ -15,6 +18,11 @@ use {
 pub enum DeployMethod {
     Local(String),
     ReleaseChannel(String),
+    Commit(
+        String, /* commit */
+        String, /* gh username */
+        String, /* repo name */
+    ),
```

Comment:
Nit: can you make this a struct for easier reading?
```suggestion
    Commit {
        commit: String,
        user_name: String,
        repo_name: String,
    },
```